### PR TITLE
Update batch filter docs with arguments

### PR DESCRIPTION
**Pull request by alexeymalihin**
Changes based on https://github.com/twigphp/Twig/blob/1.x/test/Twig/Tests/Fixtures/filters/batch_float.test

### CODE REVIEW

**Comment by SpacePossum** [comment]
Hi,
I'm sure, but I think the limit of a line is 79 characters for rst files (sphinx).
Does the spacing between 'size``:   Number'  need to be multiple spaces?

**Comment by alexeymalihin** [comment]
@SpacePossum, thanks, fixed.

**Comment by SpacePossum** [comment]
:+1:   
(I meant to type `I'm not sure`. .. .didn't mean to sound like a know all ;) )

File: doc/filters/batch.rst [hunk 1]
```diff
@@ -40,3 +40,9 @@ The above example will be rendered as:
             <td>No item</td>
         </tr>
     </table>
+
+Arguments
+---------
+
+* ``batch size``: Count in one batch. Fractional number will be rounded up.
```
Comment (fabpot) [inline doc/filters/batch.rst]: They keys here must be the ones you can use in a template, so `size` and `fill`. I'm going to fix that after merging.

**Comment by fabpot** [comment]
Thank you @alexeymalihin.

File: doc/filters/batch.rst (hunk 1 above)
Comment (fabpot) [inline doc/filters/batch.rst]: 7823014

**Pull request closed without merging**